frontend/src/lib/utils.tsx

```diff
@@ -1055,11 +1076,17 @@ export const areDatesValidForInterval = (
             parsedOldDateTo.diff(parsedOldDateFrom, 'hour') >= 2 &&
             parsedOldDateTo.diff(parsedOldDateFrom, 'hour') < 24 * 7 * 2 // 2 weeks
         )
+    } else if (interval === 'minute') {
+        return (
+            parsedOldDateTo.diff(parsedOldDateFrom, 'minute') >= 2 &&
+            parsedOldDateTo.diff(parsedOldDateFrom, 'minute') < 60 * 24 * 2 // 2 days
```

Comment:
This is 2880 minutes. Let's assume a user has the browser window sized such that the insight view is around 1000px wide – that's the case on a 14-inch MacBook – with almost 3 data points per pixel, it's not feasible to show such a dense graph. I propose 12 hours, which is going to be at the limit, with its 720 points